package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/schema",
-  "version": "1.26.0",
+  "version": "1.25.1-2",
```

Comment:
Why has the version of the schema gone backwards?

Reply:
Because this branch is older than the latest version on production